### ISSUE TITLE
Test and document Azure Files home directory restore functionality

### ISSUE THREAD

**Issue by yuvipanda** [2i2c-org/infrastructure#4426]
Backups are only useful if we know they work, and can actively restore them.

https://learn.microsoft.com/en-us/azure/backup/restore-afs?tabs=full-share-recovery has documentation on how to restore an Azure files backup.

We should test to make sure we can, and then write the smallest amount of documentation possible to make sure we can do so in times of disaster. We should rely on linking out to the cloud provider documentation as much as possible, this doesn't have to be super detailed.

In the long run, we should test these once every 6 months or so. But that's out of scope for this ticket

**Comment by sgibson91** [comment]
Hey @GeorgianaElena when you get around to this task, I created some scaffolding documentation in https://github.com/2i2c-org/infrastructure/pull/4459

**Comment by GeorgianaElena** [comment]
This is now blocked by doing https://github.com/2i2c-org/infrastructure/issues/4487 first, so I'll added this issue to the blocked column

**Comment by yuvipanda** [comment]
We discovered in #4487 we can not actually have backups for azure nfs shares.
